crates/matrix-sdk-base/src/sliding_sync.rs

```diff
@@ -358,13 +357,6 @@ impl BaseClient {
             .await?;
         }
 
-        let room_account_data = if let Some(events) = rooms_account_data.remove(room_id) {
-            self.handle_room_account_data(room_id, &events, changes).await;
```

Comment:
Are we sure account data are handled somewhere else?

Reply:
Yes, all of it is handle in [process_sliding_sync](https://github.com/matrix-org/matrix-rust-sdk/pull/3076/files#diff-25d90b0af9233646265c5c64d90f6de789fef8cf60becb897304a43a016b5730R223)